integrations/api_repo_lfs_locks_test.go

```diff
@@ -24,13 +25,13 @@ func TestAPILFSLocksNotStarted(t *testing.T) {
 	repo := models.AssertExistsAndLoadBean(t, &models.Repository{ID: 1}).(*models.Repository)
 
 	req := NewRequestf(t, "GET", "/%s/%s.git/info/lfs/locks", user.Name, repo.Name)
-	MakeRequest(t, req, http.StatusNotFound)
```

Comment:
IIRC 404 was used as to not leak info

Reply:
I used that to be in sync with the other methods:
https://github.com/go-gitea/gitea/blob/370cfde35e5b54902cf335581f15fa1d43529b43/routers/routes/web.go#L252-L279

Reply:
https://github.com/git-lfs/git-lfs/blob/main/docs/spec.md
https://docs.gitlab.com/ee/topics/git/lfs/
https://github.com/git-lfs/git-lfs/blob/main/docs/api/locking.md

404 is not spec compliant ...
we should still return 404 on unautorized private repos else follow the spec?

Reply:
It's not about the LFS spec here. `StatusForbidden` is used if a Gitea service like OpenID, Webhooks or LFS is not enabled. IMHO this should be `StatusNotFound` because a "normal" server will respond that if you request an url it doesn't know about.

Reply:
ok so just add 404 to it again ... and it's ready? @techknowlogick 

cc @KN4CK3R